kornia/augmentation/utils/__init__.py

```diff
@@ -36,4 +39,7 @@
     "_joint_range_check",
     "_singular_range_check",
     "_tuple_range_reader",
+    "_validate_input",
+    "_validate_input3D",
```

Comment:
Are those two functions used? I did not see any function called those two.

Reply:
these are decorators to check input shape for functional instead of reshape, as we discussed earlier.

Reply:
But it is not used in any functions, right?  I did a simple "Ctrl + F" here and did not see any functions are using this.

Reply:
I used these decorators in `functional.py`. For e.g., [here](https://github.com/justanhduc/kornia/blob/12c116fcbfb4073b1a87d5280a5d7dddc507086c/kornia/augmentation/functional/functional.py#L47).
I hid all the common input checks in these decorators.